send/send.c

```diff
@@ -287,6 +287,9 @@ static int edit_envelope(struct Envelope *en, SendFlags flags, struct ConfigSubs
         goto done;
     }
 
+    if (TAILQ_EMPTY(&en->to) && TAILQ_EMPTY(&en->cc) && TAILQ_EMPTY(&en->bcc))
+      goto done;
```

Comment:
Elsewhere in `send.c`, we already have an error message for this situation.

```suggestion
      {
        mutt_warning(_("No recipients specified"));
        goto done;
      }
```

Please can you downgrade that one too.
https://github.com/neomutt/neomutt/blob/b77af02186a3c800ca04ddbee9fccf9188877dee/send/send.c#L2772

The use of errors vs warnings isn't consistent yet, but my rule-of-thumb is:
- **Warning**: If we can explain to the user how to avoid the issue next time
- **Error**: Something went wrong that the user couldn't avoid happening

Reply:
Done.